### DISCUSSION TITLE
restream exisitng webradio stream

### DISCUSSION THREAD

**Discussion by sander816** [AzuraCast/AzuraCast#4719] [General]
i am trying to restream an aac stream from an existing radiostation. 
I added the url in the playlist as external playlist, but it doesn't play it .

This is the existing stream:
http://playerservices.streamtheworld.com/api/livestream-redirect/SP_R2183256_SC.mp3

**Comment by sander816** [comment]
anyone?

**Comment by paulcjkp** [comment]
Have you tried setting up a Mount Point in Azuracast, and setting up a relay in the existing stream host?
